util/types/schemas/blockchain.mol

```diff
@@ -99,3 +99,9 @@ table CellbaseWitness {
     lock:    Script,
     message: Bytes,
 }
+
+table WitnessArgs {
+    lock:                       Bytes,          // Lock args
+    type_:                      Bytes,          // Type args
```

Comment:
Just curious: have we decided to rule out the second `type` arg in case there's more than one type script in both input & output cell at the same index?